src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -17,6 +17,15 @@
         Gives X amount in the past NOT including the current second.
         "-t 1m" with now:1000 (start:940, end:999)
 
+
+    One Argument and an option
+    --------------------------
+
+    Timerange rounded
+        Gives X amount in the past NOT including the current second, rounded.
+        "-t 2m --round" with now:12:34:15 (start:12:32:00, end:12:33:59)
```

Comment:
should we add something like `--round [d|h|m|s]`? I think that for some apps `d/day` would be more relevant that just rounding by second

Reply:
ah.. it's already implemented :P ignore this